airflow/dags/sprint_3/msi_6_dag.py

```diff
@@ -0,0 +1,29 @@
+import pendulum
+from airflow import DAG
+from airflow.operators.empty import EmptyOperator
+from airflow.operators.python import PythonOperator
+
+import scripts.credentials as cr
+import scripts.main as m
+
+
+with DAG(
+        dag_id="MSI_6_dag",
+        start_date=pendulum.today(),
+        schedule_interval='0 12 * * 0-2,4-6',
+        tags=["MSI6"]
+) as dag:
+    start_op = EmptyOperator(task_id="start")
+
+    send_quote_teams_op = PythonOperator(

```

Comment:
I would propose to decompose your task to many - at this point it seems that it does Too much logic inside one unit of work